### PR TITLE
Refactored the README by moving plugin specific parts to their respective folders and added some introductory text on Android Runner

### PR DESCRIPTION
**Pull request by Maikerupitazu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,40 +166,16 @@ A JSON object to describe the profilers to be used and their arguments. Below ar
...
-    }
-  }
-```
-The frame times plugin gathers unique frame rendering durations (in nanoseconds) by utilizing `dumpsys gfxinfo framestats` and counts the amount of delayed frames that occurred following the 16ms threshold [defined by Google](https://developer.android.com/training/testing/performance).
-The sample interval is configurable but advised to keep under 120 seconds as the framestats command returns only data from frames rendered in the past 120 seconds as described [here](https://developer.android.com/training/testing/performance).
-Shorter sample intervals will not cause duplication in the frames gathered as only unique frames are kept.
-The default subject aggregation consists of combining both the frametimes as the delayed frames count in single files for easy further processing.
+| Name (quality)                                                                       | Description                                                                                                                                                                                                                                                                                 |
```
Comment (iivanoo) [inline README.md:171]: The title of the first column should be `Name (quality attribute)`

File: README.md [hunk 2]
```diff
@@ -163,40 +166,16 @@ A JSON object to describe the profilers to be used and their arguments. Below ar
...
+| [monsoon](./AndroidRunner/Plugins/monsoon/README_Monsoon.md) (Energy)                | Collects energy consumption via the Monsoon hard-ware profiler and the [Physalia tool](https://github.com/TQRG/physalia).                                                                                                                                                                   |
+| [trepn](./AndroidRunner/Plugins/trepn/README_Trepn.md) (mixed)                       | Collects data via the Trepn profiler, e.g., power consumption, battery temperature, CPUs frequency.                                                                                                                                                                                         |
+| [mem-CPU](./AndroidRunner/Plugins/android/README_Android.md) (Performance)           | Collects memory and CPU usage via the `cpuinfo` and `meminfo` Android utilities.                                                                                                                                                                                                            |
```
Comment (iivanoo) [inline README.md:176]: Add link to [adb dumpsys](https://developer.android.com/studio/command-line/dumpsys) here.

File: AndroidRunner/Plugins/trepn/README_Trepn.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Trepn Plugin
+This plugin collects data via the Trepn profiler, e.g., power consumption, battery temperature, CPUs frequency.
+
+## Configuration
+Below an example configuration is found:
+```json
+  "profilers": {
+    "trepn": {
+      "sample_interval": 100,
+      "data_points": ["battery_power", "mem_usage"]
```
Comment (iivanoo) [inline AndroidRunner/Plugins/trepn/README_Trepn.md:10]: Somewhere in the code of the plugin you will find the full list of data points supported by Trepn.

File: AndroidRunner/Plugins/trepn/README_Trepn.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Trepn Plugin
+This plugin collects data via the Trepn profiler, e.g., power consumption, battery temperature, CPUs frequency.
```
Comment (iivanoo) [inline AndroidRunner/Plugins/trepn/README_Trepn.md:2]: Here you should also mention that the Trepn plugin depends on the Trepn Android app, which is available in this folder. Then, you can move the `com.quicinc.trepn.apk' file from the `examples` folder to here.

File: README.md (hunk 1 above)
Comment (Maikerupitazu) [reply README.md:171]: Fixed it

**Comment by Maikerupitazu** [comment]
I renamed the README files, and added all configuration options I could find in the same format. I left some TODOs mostly for describing the default aggregation found in some plugins.

File: AndroidRunner/Plugins/trepn/README_Trepn.md (hunk 4 above)
Comment (Maikerupitazu) [reply AndroidRunner/Plugins/trepn/README_Trepn.md]: Added a small sentence that links to the APK's new location

File: AndroidRunner/Plugins/trepn/README_Trepn.md (hunk 3 above)
Comment (Maikerupitazu) [reply AndroidRunner/Plugins/trepn/README_Trepn.md]: Since it's quite a big list, I linked to the json itself

File: README.md (hunk 2 above)
Comment (Maikerupitazu) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (Maikerupitazu) [reply README.md]: Fixed

**Pull request merged**